config.js.template

```diff
@@ -106,4 +106,9 @@ module.exports = {
   // You should consider what components you are enabling in the /health endpoint when setting this.
   // Check the /health endpoint documentation for more info.
   considerHealthcheckWarnAsUnhealthy: true,
-};
\ No newline at end of file
+
+  // Configure default history sync mode for the wallets
+  // Accepted values are: 'polling_http_api', 'xpub_stream_ws', 'manual_stream_ws'
+  // If no value is configured the default of 'polling_http_api' will be used
+  // history_sync_mode: 'polling_http_api|xpub_stream_ws|manual_stream_ws'
```

Comment:
Should we have a list of enabled sync modes with prioritization (e.g., an integer)?

Reply:
I suggested this so a wallet configured to use `xpub_stream_ws` would fallback to `polling_http_api` if it connects to a full node where the selected mode is not available.

Reply:
This [issue] (https://github.com/HathorNetwork/hathor-wallet-lib/issues/715) was create to track this

Reply:
Ok. Thanks!